src/VisualStudio/Core/Test/DebuggerIntelliSense/CSharpDebuggerIntellisenseTests.vb

```diff
@@ -667,6 +667,29 @@ $$</Document>
             End Using
         End Sub
 
+        <ConditionalWpfFact(GetType(x86)), Trait(Traits.Feature, Traits.Features.DebuggingIntelliSense)>
+        Public Async Function TypeNumberAtStartOfViewDoesNotCrash() As Task
+            Dim text = <Workspace>
+                           <Project Language="C#" CommonReferences="true">
+                               <Document>$$</Document>
+                               <Document>class Program
```

Comment:
Why does the test scenario require two documents? If the caret is where the $$ is, what does this second file do?